scout/load/cytoband.py

```diff
@@ -23,23 +24,24 @@ def load_cytobands(resource, build, adapter):
         LOG.error(f"Resource {resource} could not be found.")
         return
 
-    with open(resource) as cytob_file:
-        for nline, line in enumerate(cytob_file):
-            # Line will look like this:
-            # 3	58600000	63800000	p14.2	gneg
-            fields = line.split("\t")
-            chrom = fields[0]
-            band = fields[3]
-
-            cytoband_obj = dict(
-                _id=generate_md5_key([build, chrom, band]),
-                band=band,
-                chrom=str(chrom),  # 3
-                start=str(int(fields[1]) + 1),  # 58600000
-                stop=str(int(fields[2]) + 1),  # 63800000
-                build=str(build),  # "37" or "38"
-            )
-            cytobands.append(cytoband_obj)
+    lines = get_file_handle(resource)
```

Comment:
:+1:

Reply:
Thanks - I saw it afterwards.. Reverting to unzipped would be totally fine with me as well.

Reply:
But, this is nice and consistent, and shows some promise to work also without --editable.